docs/products/access/topics/integration-extension/hooks/customize-token/customize-token-hook.md

```diff
@@ -0,0 +1,44 @@
+# Customize Token Web Hook
+
+The customize token web hook is meant allow access control manipulation. For security reasons, access escalation is not allowed and the
+implementor can only reduce the scopes that are associated with the resulting tokens.
+
+## Web Hook specification
+
+The web hook implementor receives multiple attributes, such as:
+
+- identity of the user (if present)
+- information about the client who was used to request the token
+- set of requested scopes
+
+Based on the above information the web hook is allowed to reduce the scopes.
+
+In case all requested scopes are reduced the Access Service will reject either, the authorization or the token flow.
+
+The web hook is executed before any access grant (authorization code or token) is created. This way, we can ensure the access to potentially
+forbidden resources will not be possible, also after refreshing the token.
+
+## API Contract
+
+The OpenAPI specification of the web hook can be found [here](./customize-token-hook.yaml)
+
+## Web Hook configuration
```

Comment:
Would be nice to mention what is the goal of the configuration and why it is needed. The connection between the configuration and the API implementation (hook implementor) should be also described.

Reply:
Good point 👍  Fixed.